ansible/roles_ocp_workloads/ocp4_workload_shared_cluster_access/tasks/post_workload.yml

```diff
@@ -1,9 +1,36 @@
 ---
 # Implement your Post Workload deployment tasks here
-# --------------------------------------------------
+#--- --------------------------------------------------
+- name: Get console route
+  k8s_info:
+    api_version: route.openshift.io/v1
+    kind: Route
+    namespace: openshift-console
+    name: console
+  register: routeconsole
 
+- name: API for CLI
+  command: oc whoami --show-server
+  register: showserver
 
+- name: Set webconsole
+  set_fact:
+    webconsole: "http://{{ routeconsole.stdout | from_json | json_query('spec.host') }}"
```

Comment:
FYI: 
Using Ansible 2.9 we can call directly to the complete location of the variable we are looking for, no more need of from_json or json_query

Just an example.

`    - k8s_info:                                                                                                                                                                           
        api_version: route.openshift.io/v1                                                                                                                                                
        kind: Route                                                                                                                                                                       
        name: grafana                                                                                                                                                                     
        namespace: openshift-monitoring                                                                                                                                                   
      register: myroute   

    - debug:                                                                                                                                                                              
        msg: "Webconsole is {{ myroute.resources[0].spec.host }}" 
`

Reply:
So if you want to change to that, it is fine. I think json_query will still have its place for more complex queries than this. 

What you have right now, though, doesn't work and it would need to be something more like this:
```
"http://{{ routeconsole | json_query('resources[].spec.host|[0]') }}"
```

Reply:
Changed to ```routeconsole.resources[0].spec.host``` tested on Ansible 2.9